### PR TITLE
Small doc update for torch_compile_debug

### PR DESCRIPTION
**Pull request by mlazos**
Updates the troubleshooting documentation with the folder structure of the debug directory

### CODE REVIEW

File: docs/source/dynamo/troubleshooting.rst [hunk 1]
```diff
@@ -447,33 +447,93 @@ calling ``torch._dynamo.utils.compile_times()`` after executing
...
-debug trace directory to be created and printed:
+Let's run an example with the following test program (repro.py):
+
+::
+
+  import torch
+  import torch._dynamo as torchdynamo
+  import torch._inductor
+
+  @torchdynamo.optimize("inductor")
```
Comment (msaroufim) [inline docs/source/dynamo/troubleshooting.rst:466]: we should probably favor torch.compile instead

File: docs/source/dynamo/troubleshooting.rst [hunk 2]
```diff
@@ -447,33 +447,93 @@ calling ``torch._dynamo.utils.compile_times()`` after executing
...
+   aot_model___0_debug.log  model__0_forward_1.0
+   $ cd model__0_forward_1.0
+   $ ls
+   debug.log  fx_graph_readable.py  fx_graph_runnable.py  fx_graph_transformed.py  ir_post_fusion.txt  ir_pre_fusion.txt  output_code.py
+
+Here is a summary of the contents:
+ - fx_graph_readable.py and fx_graph_runnable.py are the readable and runnable versions of the fx_graph received by inductor.
+ - fx_graph_transformed.py is the fx graph after inductor has run all fx passes.
+ - ir\*.txt is the inductor ir pre and post fusion.
+ - output_code.py is the compiled triton kernel for the subgraph.
```
Comment (msaroufim) [inline docs/source/dynamo/troubleshooting.rst:514]: there's also the `/tmp` folder inductor creates with the codegned kernel that's immensely helpful to look at

**Review by msaroufim — CHANGES_REQUESTED** [review]

File: docs/source/dynamo/troubleshooting.rst (hunk 2 above)
Comment (mlazos) [reply docs/source/dynamo/troubleshooting.rst:514]: These should have the same contents

**Review by msaroufim — APPROVED** [review]

**Pull request closed without merging**